### ISSUE TITLE
No rubygems release?

### ISSUE THREAD

**Issue by dbackeus** [minad/moneta#87]
Some fixes like the `Digest::Digest` deprecation has been merged to master but never released.

Anything holding you back?

**Comment by minad** [comment]
Yes, there are open issues and travis is currently not green. I need help maintaining the adapters.
Then I will make another release.

**Comment by wangyuhere** [comment]
👍

**Comment by asppsa** [comment]
Hi @minad, I think we should be good to do a new release now.

Travis is green at present.  There are still some intermittent travis failures every second commit or so, but I think they are to do with keys expiring part-way through tests when travis is under load -- I don't believe they are issues with Moneta itself.

**Comment by minad** [comment]
ok cool. I can either upload it in the next few days or add you as a gem owner if you prefer that.

**Comment by asppsa** [comment]
@minad, I'm happy to be added as a gem owner and do the new release.  What would you say the next version should be 0.9.0 or 0.8.1?   I don't see any API changes in the log, so the latter I suppose?

**Comment by minad** [comment]
I added you! I guess 0.8.1 is fine if there are no api changes.

**Comment by dbackeus** [comment]
Just a note if you're going for semver it would be nice if you could make the jump to 1.0 since anything below 1.0 is essentially for beta stages and can't properly distuinguish between breaking change / addition / patch...

After 7 years maybe it's time to accept that whatever Moneta is right now is ok and not a beta? :)
